include/seqan3/core/detail/debug_stream_byte.hpp

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Enrico Seiler <enrico.seiler AT fu-berlin.de>
+ * \brief Provides seqan3::debug_stream and related types.
+ */
+
+#pragma once
+
+#include <cstddef>
+
+#include <seqan3/core/detail/debug_stream_type.hpp>
+#include <seqan3/std/concepts>
+
+namespace seqan3
+{
+/*!\name Formatted output overloads
+ * \{
+ */
+/*!\brief A std::byte can be printed by converting it to an integer.
+ * \tparam    byte_type     The type of the input; must be equal to `std::byte`.
```

Comment:
ingroup missing?

Reply:
Then it is missing in all the files :X

Reply:
🥇

Reply:
Relates will make it appear in the documentation for the debug_stream_type so we don't need to ingroup it.